radiopadre/html/live-html-thumbnail.js

```diff
@@ -0,0 +1,45 @@
+// thanks to https://www.cambus.net/creating-thumbnails-using-systemjs-and-imagemagick/
```

Comment:
Question: So this script does thumbnails for other HTML pages?

Reply:
Yes, although this is dead code that ought to be deleted... the real working scripts (at least one of them derived from this one, if memory serves) are ``puppeteer-html-thumbail.js`` and a ``phantomjs-html-thumbail.js``

Reply:
Ok yes, the link you referred to was also a dead link